src/main/xerus/monstercat/api/Player.kt

```diff
@@ -99,6 +100,10 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	/** hides the Player and appears again displaying the latest Release */
 	fun reset() {
 		fadeOut()
+		if(Settings.PLAYEREXPORTFILE().toFile().isFile) {
```

Comment:
Either you use `Path`, and then actually use it as Path with the corresponding `Files.` method or save it as a File directly. But these back-and-forth conversions are unnecessary. I'd prefer the latter, since Path is the recommended, more modern and more compatible choice. But as soon as you convert it back to `File` all that compatibility is ruined ^^

Reply:
Oh yes, you just have to read ;) there's a helper class called `Files` to work with Paths

Reply:
4b846cf964cd3c32014cfc1ae0ffd1a01703b38c